kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/OffsetFinder.java

```diff
@@ -55,7 +55,7 @@ public void findMessages(final long timestamp, AsyncCallbacks.FindEntryCallback
         this.timestamp = timestamp;
         if (messageFindInProgressUpdater.compareAndSet(this, FALSE, TRUE)) {
             if (log.isDebugEnabled()) {
-                log.debug("[{}] Starting message position find at timestamp {}",  timestamp);
+                log.debug("Starting message position find at timestamp {}", timestamp);
```

Comment:
We could add managedLedger.getName() for the first parameter, or it would be hard for debug.
```
 log.debug("[{}] Starting message position find at timestamp {}",  managedLedger.getName(), timestamp);
```

Reply:
will fix